cmd/finch-daemon/main.go

```diff
@@ -67,6 +70,10 @@ func main() {
 	rootCmd.Flags().StringVar(&options.debugAddress, "debug-addr", "", "")
 	rootCmd.Flags().StringVar(&options.configPath, "config-file", defaultConfigPath, "Daemon Config Path")
 	rootCmd.Flags().StringVar(&options.pidFile, "pidfile", defaultPidFile, "pid file location")
+	rootCmd.Flags().StringVar(&options.regoFilePath, "rego-file", "", "Rego Policy Path")
```

Comment:
Should we call it "--opa-policy-file" . I don't know how integral rego format is to OPA, but just thinking about it, if opa supports a different policy language in the future, we will need to create a new option for the new policy language, which does not seem scalable. Sam goes for other exposed methods for Rego. Should w2e make this more generic?

Reply:
The question was more about making the user facing option generic and make it future proof. I get that we only support rego policies. I was thinking of other file formats say, yaml etc.So, we do not want to create another options calling it `--yaml-file`. If we are confident that OPA is tightly copuled with the rego format, then its fine to keep this as is.